protobuf/proto/crux.proto

```diff
@@ -614,24 +620,36 @@ message Volume {
   optional string class = 105;
 }
 
-message KeyList { repeated common.UniqueKey data = 1000; }
+message VolumeList { repeated Volume data = 1000; }
+
+message UniqueKeyList { repeated common.UniqueKey data = 1000; }
 
 message UniqueKeyValue {
   string id = 100;
   string key = 101;
   string value = 102;
 }
 
+message UniqueKeyValueList { repeated UniqueKeyValue data = 1000; }
+
+message UniqueSecretKey {
+  string id = 100;
+  string key = 101;
+  bool required = 102;
+}
+
+message UniqueSecretKeyList { repeated UniqueSecretKey data = 1000; }
+
 message UniqueSecretKeyValue {
   string id = 100;
   string key = 101;
   string value = 103;
-  string publicKey = 104;
-  bool required = 105;
-  optional bool encrypted = 106;
+  bool required = 104;
```

Comment:
Why you changed the order?

Reply:
so the optional fields would be at the end. i figure if it needs to be modified anyway, it's not a big deal to change the order as well.